mapdata/WarcraftLegacies/AbilityData/Skin/1093808193.json

```diff
@@ -43,7 +43,7 @@
     {
       "Id": 1953854049,
       "Type": 3,
-      "Value": "The Hero empowers nearby allies with the power of the mountain, increasing their armor for 30 seconds.|n|cffffcc00Level 1|r - Increases armor by 5 and attack by 20%|n|cffffcc00Level 2|r - Increases armor by 6 and attack by 40%|n|cffffcc00Level 3|r - Increases armor by 7 and attack by 60%|n|cffffcc00Level 4|r - Increases armor by 8 and attack by 80%"
+      "Value": "The Hero empowers nearby allies with the power of the mountain, increasing their armor for 30 seconds.|n|cffffcc00Level 1|r - Increases armor by 5 and attack by 10%|n|cffffcc00Level 2|r - Increases armor by 6 and attack by 20%|n|cffffcc00Level 3|r - Increases armor by 7 and attack by 30%|n|cffffcc00Level 4|r - Increases armor by 8 and attack by 40%"
```

Comment:
Ok, I think since this tooltip was wrong for a while we just need to bite the bullet and use data fields in the tooltip instead of writing the numbers out so that can't happen again.

If you're not familiar with data fields you can check out a tutorial (e.g. https://www.hiveworkshop.com/threads/raw-data-tutorial.32324/) or just base it off one of the existing vanilla tooltips that use them. If you can't figure it out, ping me (or Sebas).